components/dialog/demo/dialog-fullscreen.html

```diff
@@ -42,6 +44,66 @@ <h2>Fullscreen Dialog</h2>
 						</d2l-dropdown-menu>
 					</d2l-dropdown-more>
 					<d2l-dialog-fullscreen id="dialogFullscreen" title-text="Fullscreen Title">
+						<p>Deadlights jack lad schooner scallywag dance the hempen jig carouser broadside cable strike colors. Bring a spring upon her cable holystone blow the man down spanker</p>
+						<p>Shiver me timbers to go on account lookout wherry doubloon chase. Belay yo-ho-ho keelhaul squiffy black spot yardarm spyglass sheet transom heave to.</p>
+						<p>Trysail Sail ho Corsair red ensign hulk smartly boom jib rum gangway. Case shot Shiver me timbers gangplank crack Jennys tea cup ballast Blimey lee snow crow's nest rutters. Fluke jib scourge of the seven seas boatswain schooner gaff booty Jack Tar transom spirits.</p>
+						<p>Deadlights jack lad schooner scallywag dance the hempen jig carouser broadside cable strike colors. Bring a spring upon her cable holystone blow the man down spanker</p>
+						<p>Shiver me timbers to go on account lookout wherry doubloon chase. Belay yo-ho-ho keelhaul squiffy black spot yardarm spyglass sheet transom heave to.</p>
+						<p>Trysail Sail ho Corsair red ensign hulk smartly boom jib rum gangway. Case shot Shiver me timbers gangplank crack Jennys tea cup ballast Blimey lee snow crow's nest rutters. Fluke jib scourge of the seven seas boatswain schooner gaff booty Jack Tar transom spirits.</p>
+						<d2l-button slot="footer" primary data-dialog-action="save">Save</d2l-button>
+						<d2l-button slot="footer" data-dialog-action>Cancel</d2l-button>
+					</d2l-dialog-fullscreen>
+					<script>
+						document.querySelector('#open').addEventListener('click', () => {
+							document.querySelector('#dialogFullscreen').opened = true;
+						});
+						document.querySelector('#dropdown').addEventListener('click', () => {
+							document.querySelector('#dialogFullscreen').opened = true;
+						});
+						document.querySelector('#dialogFullscreen').addEventListener('d2l-dialog-close', (e) => {
+							console.log('confirm action:', e.detail.action);
+						});
+					</script>
+				</template>
+			</d2l-demo-snippet>
+
+			<h2>Fullscreen Dialog with Content Overflow</h2>
+
+			<d2l-demo-snippet>
+				<template>
+					<d2l-button id="openOverflow">Show Dialog</d2l-button>
+					<d2l-dropdown-more text="Open!">
+						<d2l-dropdown-menu>
+							<d2l-menu label="Dialog">
+								<d2l-menu-item id="dropdownOverflow" text="See dialog"></d2l-menu-item>
+							</d2l-menu>
+						</d2l-dropdown-menu>
+					</d2l-dropdown-more>
+					<d2l-dialog-fullscreen id="dialogFullscreenOverflow" title-text="Fullscreen Title">
```

Comment:
Added another demo here with a long tab list and overflowing text.

Reply:
Would adding this as a visual diff test instead (or in addition) maybe help us catch a regression to this in the future?

Reply:
We do have tests for the vertical overflow. I'll add one with the tabs to make sure we have horizontal overflow as well.